pcsx2/VMManager.cpp

```diff
@@ -573,6 +573,9 @@ bool VMManager::ApplyBootParameters(const VMBootParameters& params)
 {
 	const bool default_fast_boot = Host::GetBoolSettingValue("EmuCore", "EnableFastBoot", true);
 	EmuConfig.UseBOOT2Injection = params.fast_boot.value_or(default_fast_boot);
+	EmuConfig.PatchBios = Host::GetBoolSettingValue("EmuCore", "PatchBios", false);
```

Comment:
No need to load the setting here, since it's part of the config (and will get loaded in `Pcsx2Config`).